data/core/rootview.lua

```diff
@@ -798,14 +799,25 @@ function RootView:on_mouse_pressed(button, x, y, clicks)
 end
 
 
-function RootView:on_mouse_released(...)
+function RootView:on_mouse_released(button, x, y)
   if self.dragged_divider then
     self.dragged_divider = nil
   end
   if self.dragged_node then
+    local _, _, _, h = Node.get_tab_rect(self.dragged_node[1], self.dragged_node[2])
+    if x < 0 or x > self.size.x or y < 0 or y > h then
```

Comment:
If we are in a `View` _not_ in the top row, even just clicking on a tab splits it.
What if we only split the tab if it is dragged outside the window?

Reply:
I was trying to mimick Chrome's behavior, but clearly it didn't work too well in this situation. I'll change it.

Reply:
I think I fixed it. You might wanna check it out again.